etc/ciao/ciao.json

```diff
@@ -0,0 +1,9 @@
+{
+    "server": "localhost3",
+    "cacert": "/etc/pki/ciao/CAcert-server-localhost.pem",
+    "cert": "/etc/pki/ciao/cert-client-localhost.pem",
```

Comment:
A ciao-cert generated private key name will contain the embedded role. In that case this would be: cert-CNAgent-localhost.pem. Please use that name as well.